src/ui/simulator/windows/message.h

```diff
@@ -92,6 +92,8 @@ class Message final : public wxDialog
         btnUpgrade,
         //! Standard button: continue
         btnContinue,
+        //! Standard button: view simulation results
+        btnViewResults,
```

Comment:
This change is ok for me.
It's just that **pReturnStatus** should not be defined as a **uint**, but more as a item of an **enum class**, otherwise one can't swiftly know it's related to an (old) **enum**

Reply:
Done. I cleaned up the code and added a template to remove an ugly double cast (`void*` -> `size_t` -> `uint`).